packages/components/src/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -6,35 +6,64 @@ import { translate } from 'react-i18next';
 
 import theme from './Breadcrumbs.scss';
 import { Action, ActionDropdown } from '../Actions';
-import Icon from '../Icon';
+import Skeleton from '../Skeleton/Skeleton.component';
 import I18N_DOMAIN_COMPONENTS from '../constants';
 import getDefaultT from '../translate';
 
 /**
- * Default max items to display without starting by ellipsis
+ * Default max items to display
+ * @type {number}
  */
-const DEFAULT_MAX_ITEMS = 3;
+const DEFAULT_MAX_ITEMS = 4;
 
 /**
- * Indicate the current page's location within a navigational hierarchy.
+ * Default number of items before adding an ellipsis
+ * @type {number}
+ */
+const DEFAULT_NB_ITEMS_BEFORE_ELLIPSIS = 1;
+
+/**
+ * Indicate the current page location within a navigational hierarchy.
  * @param {object} props   react props
  * @example
  <Breadcrumbs
- maxItems={3}
+ maxItems={4}
  items={items}
  />
  */
-export function BreadcrumbsComponent(props) {
-	const nbItems = props.items.length;
-	const maxItemsToDisplay = props.maxItems;
-	const maxItemsReached = nbItems > maxItemsToDisplay;
-	const ellipsisIndex = nbItems - 1 - maxItemsToDisplay;
-	const hiddenItems = props.items.slice(0, ellipsisIndex + 1).map((hiddenItem, index) => ({
-		id: `${props.id}-item-${index}`,
-		label: hiddenItem.text,
-		title: hiddenItem.title,
-		onClick: event => hiddenItem.onClick(event, hiddenItem),
-	}));
+export function BreadcrumbsComponent({ loading, id, items, maxItems, t }) {
+	if (loading) {
+		return (
+			<div
+				className={classNames(
+					theme['tc-breadcrumb'],
+					theme.loading,
+					'tc-breadcrumb',
+					'tc-breadcrumb--loading',
+				)}
+			>
+				{[
+					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
```

Comment:
I prefer use a constant to set all skeleton and map on this for lisibility.
```suggestion
const BREADCRUMB_SKELETONS = [
					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
					{ size: Skeleton.SIZES.small, type: Skeleton.TYPES.circle },
					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
				]
				
				BREADCRUMB_SKELETONS.map...

```